backend/src/main/java/gov/cdc/usds/simplereport/api/apiuser/ApiUserMutationResolver.java

```diff
@@ -65,11 +57,11 @@ public User updateUser(
         ApiUser apiUser = _us.updateUser(newEmail, oldEmail, firstName, middleName, lastName, suffix);
         Optional<Organization> org = _os.getOrganizationForUser(apiUser);
         Boolean isAdmin = _us.isAdminUser(apiUser);
-        return new User(apiUser, org, isAdmin, getDefaultPermissions());
+        return new User(apiUser, org, isAdmin, getDefaultRole());
     }
 
-    private List<UserPermission> getDefaultPermissions() {
-        return new ArrayList<>(OrganizationRole.USER.getGrantedPermissions());
+    private Optional<OrganizationRole> getDefaultRole() {
```

Comment:
heads up that i'm planning to do away with this method in a future PR anyway, but i'm not opposed to this change for now.

Reply:
Yeah, having it actually reflect the user's granted roles would clearly be better—this is just there to make sure we return a valid object.